apps/dotcms-ui/src/app/api/services/add-to-bundle/add-to-bundle.service.ts

```diff
@@ -50,14 +50,14 @@ export class AddToBundleService {
         bundleData: DotBundle
     ): Observable<DotAjaxActionResponseView> {
         return this.coreWebService
-            .request({
+            .requestView({
```

Comment:
You need to test the sh*t out of this change because it could lead to problems. I wouldn't recommend doing this.

Reply:
Noted it!